src/Microsoft.ML.Tokenizers/PACKAGE.md

```diff
@@ -30,12 +30,12 @@ string source = "Text tokenization is the process of splitting a string into a l
 Console.WriteLine($"Tokens: {tokenizer.CountTokens(source)}");
 // prints: Tokens: 16
 
-var trimIndex = tokenizer.GetIndexByTokenCountFromEnd(source, 5, out string processedText, out _);
-Console.WriteLine($"5 tokens from end: {processedText.Substring(trimIndex)}");
+var trimIndex = tokenizer.GetIndexByTokenCountFromEnd(source, 5, out string normalizedText, out _);
+Console.WriteLine($"5 tokens from end: {source.Substring(trimIndex)}");
```

Comment:
Need to write it as:

Console.WriteLine($"5 tokens from end: {(normalizedText ?? source).Substring(trimIndex)}");

please fix the other on in line 38